mongodb/src/main/scala/akka/stream/alpakka/mongodb/scaladsl/MongoSource.scala

```diff
@@ -0,0 +1,14 @@
+package akka.stream.alpakka.mongodb.scaladsl
+
+import akka.stream.scaladsl.Source
+import org.mongodb.scala.{Document, Observable}
+import play.api.libs.json.{JsObject, Json}
+import akka.stream.alpakka.mongodb.Implicits._
+
+object MongoSource {
+
+  def apply(query: Observable[Document]) ={
+    Source.fromPublisher(query).map(doc=>Json.parse(doc.toJson()).as[JsObject])
```

Comment:
Make parsing to JsObject optional (if it should be here at all), also, always explicitly define return type of public api methods.

What about removing the json parsing from the connector but providing a sample of how to use `play-json` in the docs?